kahuna/public/js/components/gu-lazy-gallery-shortcuts/gu-lazy-gallery-shortcuts.js

```diff
@@ -27,13 +27,13 @@ lazyGalleryShortcuts.directive('guLazyGalleryShortcuts',
                     combo: 'left',
                     description: 'Go to the previous image',
                     allowIn: ['INPUT'],
-                    callback: invoke('previousImage')
+                    callback: invoke('previousItem')
```

Comment:
`Item` not `Image` - very prudent: in preparation of Grid support for video ;-) Absolutely innocent ping: @akash1810.